tests/cli/test_launch.py

```diff
@@ -127,3 +127,19 @@ def test_plot_eos_print_table(run_cli_command, generate_eos_node, include_magnet
     options = [str(node.pk), '--print-table']
     result = run_cli_command(cmd_plot_eos, options)
     data_regression.check({'output_lines': result.output_lines})
+
+
+@pytest.mark.usefixtures('aiida_profile')
+def test_relax_magn_per_type(run_cli_command, generate_structure, generate_code):
+    """Test the `--magnetization-per-site` option."""
+    structure = generate_structure()
+    structure.append_atom(position=(0.000, 0.000, 0.468), symbols=['H'])
+    structure.append_atom(position=(0.000, 0.000, 0.268), symbols=['H'])
+    structure.store()
+    generate_code('quantumespresso.pw').store()
+
+    # Correct
```

Comment:
The following inputs are actually incorrect and you don't have a test where it _is_ correct

Reply:
Yes, you are right. I think I meant to add a test where all is correct, but I then saw that this is not present for all the others "options" of the CLI.